manifests/base/config/burrito-controllers-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/part-of: burrito
```

Comment:
Missing labels `app.kubernetes.io/component: controllers` and `app.kubernetes.io/name: burrito-controllers`